### DISCUSSION TITLE
续费了两年现在提示账号过期还是免费版

### DISCUSSION THREAD

**Discussion by alala1314** [getlantern/lantern-forum-cn#798] [Q&A]
邮箱416043995@qq.com 支付宝支付
请帮忙看看
![微信图片_20240820011138](https://github.com/user-attachments/assets/8d226d68-b3b6-49b7-a6e4-f41b4658c39a)

**Comment by MaxMKMax** [comment]
@alala1314 您好. 已经帮您提交问题上去了. 谢谢您支持蓝灯!

**Reply by alala1314** [reply]
> @alala1314 您好. 已经帮您提交问题上去了. 谢谢您支持蓝灯!

有结果了吗

**Reply by alala1314** [reply]
> @alala1314 您好. 已经帮您提交问题上去了. 谢谢您支持蓝灯!

现在有结果了吗 @Mike777Khoo

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/ff747302-cbbe-45dd-a4b6-09166553f8d9)

@alala1314 您好. 已经给您弄好了.

**Reply by alala1314** [reply]
@Mike777Khoo 谢谢 但是说送的三个月没了呢
